### ISSUE TITLE
Uncaught (in promise) Error: Attempted to update component `React_time_ago` that has already been unmounted (or failed to mount)

### ISSUE THREAD

**Issue by ggwc82** [halt-hammerzeit/react-time-ago#3]
Hi,

I'm getting this error when I'm trying to get this working in my app:

`invariant.js:44 Uncaught (in promise) Error: Attempted to update component `React_time_ago` that has already been unmounted (or failed to mount).`

I have ensured to include the following in my app.js file:

```
import react_time_ago from 'react-time-ago';
import javascript_time_ago from 'javascript-time-ago'
javascript_time_ago.locale(require('javascript-time-ago/locales/en'))
javascript_time_ago.locale(require('javascript-time-ago/locales/ru'))

require('javascript-time-ago/intl-messageformat-global')
require('intl-messageformat/dist/locale-data/en')
require('intl-messageformat/dist/locale-data/ru')
```
However, the problem remains. Any idea on what this could be due to?

**Comment by halt-hammerzeit** [comment]
I guess this was due to the refresh timer trying to update an unmounted component:
https://github.com/halt-hammerzeit/react-time-ago/blob/89179957b67c5fc3d64da67a2f25a0e0877a5278/source/react%20time%20ago.js#L207
Not sure why it happened though, the code takes precautions to unregister components being unmounted.

I performed some refactoring of the library today.
Try the new version.
Not necessarily will it fix your issue but perhaps it might.
